src/index.js

```diff
@@ -0,0 +1,212 @@
+  ['`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace'],
+  ['Tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\', 'Del'],
+  ['Caps Lock', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', '\'', 'Enter'],
+  ['Shift', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/', '↑', 'Shift'],
+  ['Ctrl', 'Win', 'Alt', ' ', 'Alt', '←', '↓', '→', 'Ctrl'],
+];
+
+const symbolsEventCode = [
+  ['Backquote', 'Digit1', 'Digit2', 'Digit3', 'Digit4', 'Digit5', 'Digit6', 'Digit7', 'Digit8', 'Digit9', 'Digit0', 'Minus', 'Equal', 'Backspace'],
+  ['Tab', 'KeyQ', 'KeyW', 'KeyE', 'KeyR', 'KeyT', 'KeyY', 'KeyU', 'KeyI', 'KeyO', 'KeyP', 'BracketLeft', 'BracketRight', 'Backslash', 'Delete'],
+  ['CapsLock', 'KeyA', 'KeyS', 'KeyD', 'KeyF', 'KeyG', 'KeyH', 'KeyJ', 'KeyK', 'KeyL', 'Semicolon', 'Quote', 'Enter'],
+  ['ShiftLeft', 'KeyZ', 'KeyX', 'KeyC', 'KeyV', 'KeyB', 'KeyN', 'KeyM', 'Comma', 'Period', 'Slash', 'ArrowUp', 'ShiftRight'],
+  ['ControlLeft', 'Windows', 'AltLeft', 'Space', 'AltRight', 'ArrowLeft', 'ArrowDown', 'ArrowRight', 'ControlRight'],
+];
+
+const symbolsShift = [
+  ['Ё', '!', '"', '№', ';', '%', ':', '?', '*', '(', ')', '_', '+', 'Backspace'],
+  ['Tab', 'Й', 'Ц', 'У', 'К', 'Е', 'Н', 'Г', 'Ш', 'Щ', 'З', 'Х', 'Ъ', '/', 'Del'],
+  ['Caps Lock', 'Ф', 'Ы', 'В', 'А', 'П', 'Р', 'О', 'Л', 'Д', 'Ж', 'Э', 'Enter'],
+  ['Shift', 'Я', 'Ч', 'С', 'М', 'И', 'Т', 'Ь', 'Б', 'Ю', ',', '↑', 'Shift'],
+  ['Ctrl', 'Win', 'Alt', ' ', 'Alt', '←', '↓', '→', 'Ctrl'],
+];
+
+// const engSymbolsShift = [
+//   ['~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '+', 'Backspace'],
+//   ['Tab', 'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P', '{', '}', '|', 'Del'],
+//   ['Caps Lock', 'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', ':', '"', 'Enter'],
+//   ['Shift', 'Z', 'X', 'C', 'V', 'B', 'N', 'M', '<', '>', '?', '↑', 'Shift'],
+//   ['Ctrl', 'Win', 'Alt', ' ', 'Alt', '←', '↓', '→', 'Ctrl'],
+// ];
+
+
+const textarea = document.createElement('textarea');
+document.body.append(textarea);
+textarea.className = 'textarea';
+
+const keyboardBlock = document.createElement('div');
+keyboardBlock.className = 'keyboard';
+document.body.append(keyboardBlock);
+
+const numberOfKeys = [14, 15, 13, 13, 9];
+
+const key = document.createElement('div');
+
+for (let i = 0; i < 5; i += 1) {
+  const row = document.createElement('div');
+  row.className = 'row';
+  keyboardBlock.appendChild(row.cloneNode(true));
+}
+
+const rows = keyboardBlock.querySelectorAll('.row');
+let i = 0;
+
+rows.forEach((r) => {
+  for (let l = 0; l < numberOfKeys[i]; l += 1) {
+    key.className = 'key';
+    r.append(key.cloneNode(true));
+  }
+  i += 1;
+});
+
+const keys = keyboardBlock.querySelectorAll('.key');
+
+let r = 0;
+let c = 0;
+
+keys.forEach((k) => {
+  k.setAttribute('id', symbolsEventCode[r][c]);
+  k.innerHTML = symbols[r][c];
+  c += 1;
+  if (c === symbols[r].length) {
+    r += 1;
+    c = 0;
+  }
+
+  k.onclick = function click() {
```

Comment:
Use addEventListener instead of onclick. Also it's not a good idea to assign listener to the each button. You can create only 1 listener to the container and use delegating.